Extensions/Content/Dnn.PersonaBar.Extensions/WebApps/Licensing.Web/src/components/platform/index.jsx

```diff
@@ -1,91 +1,101 @@
 import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import {
+    licensing as LicensingActions
+} from "../../actions";
 import "./style.less";
 import resx from "../../resources";
 import styles from "./style.less";
 
 /*eslint-disable quotes*/
-const starIcon = require(`!raw-loader!./../svg/star_circle.svg`).default;
-const evoqIcon = require(`!raw-loader!./../svg/evoq.svg`).default;
-const infoIcon = require(`!raw-loader!./../svg/info_circle.svg`).default;
+const dnnTechnologyEditorialIcon = require(`!raw-loader!./../svg/dnn_technology_editorial.svg`).default;
+const githubIcon = require(`!raw-loader!./../svg/github.svg`).default;
+const dnnIcon = require(`!raw-loader!./../svg/dnn_logo_primary.svg`).default;
+const docsIcon = require(`!raw-loader!./../svg/dnn_docs_logo.svg`).default;
 
 class Platform extends Component {
     constructor() {
         super();
     }
 
-    renderHeader() {
-        let tableFields = [];
-        tableFields.push({ "name": resx.get("LicenseType.Header"), "id": "LicenseType" });
-        tableFields.push({ "name": resx.get("InvoiceNumber.Header"), "id": "InvoiceNumber" });
-        tableFields.push({ "name": resx.get("WebServer.Header"), "id": "WebServer" });
-        tableFields.push({ "name": resx.get("Activated.Header"), "id": "Activated" });
-        tableFields.push({ "name": resx.get("Expires.Header"), "id": "Expires" });
-
-        let tableHeaders = tableFields.map((field) => {
-            let className = "header-" + field.id;
-            return <div className={className} key={"header-" + field.id}>
-                <span>{field.name}</span>
-            </div>;
-        });
-        return <div className="licenses-header-row">{tableHeaders}</div>;
+    renderVersion() {
+        return <div className="intro-header-row">{this.props.productVersion}</div>;
     }
 
-    onEvoqClick() {
-        window.open("http://www.dnnsoftware.com/cms-features", "_blank");
+    onGitHubClick() {
+        window.open("//github.com/dnnsoftware/Dnn.Platform", "_blank");
```

Comment:
Should we add https to these links since they are external so we avoid a redirect from http to https ?

Reply:
Okay, yes...after a bit of research, it appears using protocol-relative URLs is an anti-pattern.  I will push a new commit with this change shortly.  Thanks @valadas!

Reply:
Just pushed a new commit to address this concern.